spec/std/array_spec.cr

```diff
@@ -2041,4 +2041,18 @@ describe "Array" do
       ary.skip(-1)
     end
   end
+
+  it "#resize" do
```

Comment:
This includes 2 different test cases.
Could we split this up.  I look through the specs sometimes when something breaks and it is not always obvious what it is trying to test.